### ISSUE TITLE
[Bug] [Quest] Prorite's Money

### ISSUE THREAD

**Issue by jinnex** [Yoress/friendly-dollop#127]
**Quest ID: 1319**

Issue: Quest finisher triggered without talking to Prorite. I talked to the last NPC Mappinerk (ID: 205240), and finisher triggered right after for some reason.

http://aiondatabase.net/en/quest/1319/

**Comment by Yoress** [comment]
Is this something that would stand out when trying to complete the quest? What's supposed to happen, instead?

**Comment by jinnex** [comment]
Yeah. You're supposed to talk to Prorite, and then quest finishes. But for some reason dialogue appears automatically after talking with Mappinerk.
